build.sh

```diff
@@ -9,5 +9,13 @@ BROWSERIFY_CMD="browserify -s detectEthereumProvider"
 mkdir -p ./dist
 rm -rf ./dist/*
 
-$BROWSERIFY_CMD src/index.js > dist/detect-provider.js
-$BROWSERIFY_CMD -p tinyify src/index.js > dist/detect-provider.min.js
+tsc --project . --outDir dist
+
+if [[ "${OSTYPE}" == "darwin"* ]]; then
+  sed -i '' 's/^export default/module.exports =/g' 'dist/index.js'
+else
+  sed -i 's/^export default/module.exports =/g' 'dist/index.js'
+fi
```

Comment:
We should be able to replace this logic with the correct module syntax for CommonJS in the source files. Can we use [`export =`](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) instead?

Reply:
This is what I was looking for! 🙏 

02d2728